website/content/docs/concepts/filtering/audit.mdx

```diff
@@ -0,0 +1,165 @@
+---
+layout: docs
+page_title: Audit Filtering
+description: |-
+  How to filter audit entries in Vault.
```

Comment:
Is this actually a how to guide or a concept document? It's written as a concept doc (and under the concept section), but the description makes me think it's supposed to be a how-to? Let me know which you intended and I'll make edits appropriate to the doc type :)

Reply:
It's supposed to cover the concept of filtering specific to audit, with examples. I think that means the description isn't right. 😞

Reply:
Okay, I'll edit it as a concept doc and suggest a new description for you :)